src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -338,11 +321,20 @@ void ViewSceneParse(HandlerContext & ctx, const CommandData & req, GroupDataProv
 
     response.status = to_underlying(Protocols::InteractionModel::Status::Success);
 
-    // Verify if name was saved in the scene
-    if (scene.mStorageData.mNameLength)
+    // Verifies how to convert transition time
+    if (scene.mStorageData.mSceneTransitionTimeMs)
```

Comment:
Why are we skipping this step if mSceneTransitionTimeMs is 0?  0 is a totally valid time we should be returning.